app/src/main/java/org/simple/clinic/analytics/Analytics.kt

```diff
@@ -33,7 +33,14 @@ object Analytics {
         val uuidString = uuid.toString()
 
         setUserIdentity(uuidString)
-        setProperty("userId", uuidString)
+      }
```

Comment:
I see this is not part of your commits, but Android Studio IDE warns us that:
![image](https://user-images.githubusercontent.com/8101796/58401563-d981b580-807b-11e9-9ef9-95c766ef0e47.png)

Can you please fix this on lines 19 and 27?

Reply:
Done.